lib/matplotlib/patches.py

```diff
@@ -3505,11 +3513,13 @@ class CurveFilledB(_Curve):
 
         def __init__(self, head_length=.4, head_width=.2):
             """
-            *head_length*
-              length of the arrow head
+            Parameters
+            ----------
+            head_length : float, optional, default : .4
```

Comment:
Optional does not imply that passing `None` does the default; it implies that not passing it uses the default.